gateleen-kafka/README_kafka.md

```diff
@@ -22,18 +22,16 @@ The following example shows a topic configuration resource with configurations f
 bottom. The first topic name matching the provided topic is used.
 ```json
 {
-  "topics": {
-    "my.topic.*": {
-      "bootstrap.servers": "localhost:9092",
-      "key.serializer": "org.apache.kafka.common.serialization.StringSerializer",
-      "value.serializer": "org.apache.kafka.common.serialization.StringSerializer"
-    },
-    ".+": {
-      "bootstrap.servers": "localhost:9093",
-      "key.serializer": "org.apache.kafka.common.serialization.StringSerializer",
-      "value.serializer": "org.apache.kafka.common.serialization.StringSerializer"
-    }
-  }
+	"my.topic.*": {
```

Comment:
Not sure if its a good idea having a variable key at top level. This means its not possible to add anything else to this json at top level without conflicting the variable keys.

Or is this just fixing doc to how it actually is implemented? (*couldn't find related changes elsewhere*)

Reply:
Thanks for the remark. It's just fixing the documentation.

Reply:
Ok :) Then sounds fine 👍